add-on/panels/sidebar.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
```

Comment:
Oh now I am confused, this seems to be a replacement for the conversation window - I think this file needs to be a `git mv` not a copy.